boards/main/Cargo.toml

```diff
@@ -15,6 +15,4 @@ sdio-host = "0.9.0"
 defmt-rtt = "0.4.0"
 defmt = "0.3.2"
 atsamd-hal = { workspace = true }
-#
-#[package.metadata.docs.rs]
-#targets = ["thumbv7em-none-eabihf"]
```

Comment:
I can't see anywhere else that this is added and I think we still want it for local documentation, can you maybe add this line to 'common-arm'?

Reply:
Do we need it? In my experience, the docs are building fine without this line

Reply:
Just built the docs locally, looks like you're correct. I'm fine with removing it then.